com.playeveryware.eos/Runtime/Core/Config/EOSConfig.cs

```diff
@@ -335,6 +336,14 @@ protected EOSConfig() : base("EpicOnlineServicesConfig.json")
             "to the SDK.", 4)]
         public bool hackForceSendInputDirectlyToSDK;
 
+        [ConfigField("Default Activate Overlay Button",
+            ConfigFieldType.Enum,
+            "Users can press the buttons associated with this value to activate " +
+            "the Epic Social Overlay. If this is multiple buttons, the user " +
+            "presses them in combination to activate the Epic Social Overlay. " +
+            "Platforms without a custom override will use this value.", 4)]
+        public InputStateButtonFlags defaultActivateOverlayButtonCombination;
```

Comment:
Because of how `InputStateButtonFlags` is implemented - it may be clearer to name this `activatingOverlayInput`, the type being `InputStateButtonFlags` implies that the value can be the result of a bitwise operation on the input state button flags enum.

**However**, _separate_ from this being what I believe the correct implementation _should_ be - we are currently limited to the functionality provided by Unity's `JsonUtility`, because these values need to be serializable. 

This means that the best data type to use is `List<string>`. If the field stored a non-flaggable `enum`, then the field could be a `string`, but because the intent is to provide the ability to define a _set_ of buttons that activate the overlay, it needs to be a `List`. Some more work will need to be done on the rendering side to support this.